core/src/test/scala/org/genivi/sota/core/PackageUploadSpec.scala

```diff
@@ -48,18 +44,10 @@ class PackageUploadSpec extends PropSpec
 
   class Service(resolverResult: Future[Unit] ) {
```

Comment:
`resolverResult` is not used anymore, so could remvove, and make `Service` an object